### ISSUE TITLE
[Bug]: macos 15 alert

### ISSUE THREAD

**Issue by yonatankremer** [jordanbaird/Ice#280]
### Check Existing Issues

- [X] I have searched existing issues, and this issue is not a duplicate

### Description

When clicking on the Ice menu bar icon after... a while, I'd guess it's somewhere around 30 minutes, a macos alert will continuously pop up, saying Ice is recording the display, asking for re-approval. It's only after clicking approve that Ice reacts. No further bugs spotted so far though :)

Will screenshot the next time it happens.

### Steps to Reproduce

Quite unclear, stated above

### Ice Version

0.10.3

### macOS Version

15.0

**Comment by jordanbaird** [comment]
Duplicate of #170
